backend/src/federation/client/Client_1_0.ts

```diff
@@ -0,0 +1,46 @@
+import { FederatedCommunity as DbFederatedCommunity } from '@entity/FederatedCommunity'
+import { GraphQLClient } from 'graphql-request'
+
+import { getPublicKey } from '@/federation/query/getPublicKey'
+import { LogError } from '@/server/LogError'
+import { backendLogger as logger } from '@/server/logger'
+
+// eslint-disable-next-line camelcase
+export class Client_1_0 {
+  dbCom: DbFederatedCommunity
+  endpoint: string
+  client: GraphQLClient
+
+  constructor(dbCom: DbFederatedCommunity) {
+    this.dbCom = dbCom
+    this.endpoint = `${dbCom.endPoint.endsWith('/') ? dbCom.endPoint : dbCom.endPoint + '/'}${
+      dbCom.apiVersion
+    }/`
+    this.client = new GraphQLClient(this.endpoint, {
+      method: 'GET',
+      jsonSerializer: {
+        parse: JSON.parse,
+        stringify: JSON.stringify,
+      },
+    })
+  }
+
+  getPublicKey = async (): Promise<string | undefined> => {
+    logger.info(`requestGetPublicKey with endpoint='${this.endpoint}'...`)
+    try {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      const { data, errors, headers, status } = await this.client.rawRequest(getPublicKey, {})
+      logger.debug(`Response-Data:`, data, errors, headers, status)
```

Comment:
Don't know if you need the ` could be done with '
```suggestion
      logger.debug(`Response-Data: ${data} ${errors} ${headers} ${status}`)
```
or
```suggestion
      logger.debug('Response-Data:', data, errors, headers, status)
```